packages/jest-bolt-receiver/README.md

```diff
@@ -0,0 +1,80 @@
+# Jest Bolt Receiver
+
+A [receiver](https://slack.dev/bolt/concepts#receiver) built for testing Slack Bolt apps with Jest. This receiver allows you to easily send mock events to your app and write consistent tests.
```

Comment:
Having a better description of what you had to do when testing before this package and after would help demonstrate the value more.